src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -204,6 +204,8 @@ public function toolbarAction(Request $request, $token)
             $url = $this->generator->generate('_profiler', array('token' => $token));
         } catch (\Exception $e) {
             // the profiler is not enabled
+        } catch (\Throwable $e) {
+            // the profiler is not enabled
```

Comment:
that's wrong. the router throws an exception when the route is not defined, so there is no need to catch Throwable here except to make debugging harder (we could actually be even more specific in the catched exception by catching only router exceptions AFAICT)